compiler/luci/pass/src/helpers/LayerInfoMap.cpp

```diff
@@ -39,6 +149,20 @@ LayerInfoMap layer_info_map(loco::Graph *g, std::vector<LayerInfo> &layers_info)
 
       if (cnode->name() == name)
       {
+        // Check and add multiple-output node and its outputs to info_by_name
+        if (is_multiple_output_node(cnode))
+        {
+          add_multi_output_node(info_by_name, info, cnode);
+          found = true;
+          continue;
+        }
+        if (auto multi_output = get_multi_output_node(cnode))
+        {
+          add_multi_output_node(info_by_name, info, multi_output);
+          found = true;
+          continue;
+        }
+
```

Comment:
It seems that we can merge these  two if statements.  Could you remove the first if statement and `get_multi_output_node` returns `multi_output` node if cnode is multi output node? i.e., the first if statement is moved to `get_multi_output_node`.